cartography/intel/gcp/util/common.py

```diff
@@ -5,7 +5,7 @@
 
 def parse_and_validate_gcp_requested_syncs(gcp_requested_syncs: str) -> List[str]:
     validated_resources: List[str] = []
-    for resource in gcp_requested_syncs.split(','):
+    for resource in gcp_requested_syncs:
```

Comment:
`services` parameters would be a comma separated string, not an array of strings. Let's use the split function here.